hc/api/tests/test_sendalerts.py

```diff
@@ -37,6 +38,6 @@ def test_it_handles_grace_period(self):
         check.save()
 
         # Expect no exceptions--
-        Command().handle_one(check)
+        self.assertTrue(Command().handle_one(check))
```

Comment:
There was no need to modify this one. The command would return an exception if there was an issue. So the test is by default passing if the command does not return an exception. That's what the comment above was saying

Reply:
Okay.  I also noticed it and have resolved it. Thanks for the feedback on that.